### PR TITLE
controller service add identity service sanity test

### PR DESCRIPTION
**Pull request by carlory**
**What type of PR is this?**

> /kind feature

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
controller service add identity service sanity test
```

### CODE REVIEW

File: pkg/sanity/identity.go [hunk 1]
```diff
@@ -32,80 +32,88 @@ import (
...
 	var (
-		c csi.IdentityClient
+		cs map[string]csi.IdentityClient
 	)
 
 	BeforeEach(func() {
-		c = csi.NewIdentityClient(sc.Conn)
+		cs = make(map[string]csi.IdentityClient)
+		cs["Node Service"] = csi.NewIdentityClient(sc.Conn)
+		if sc.ControllerConn != sc.Conn {
```
Comment (xing-yang) [inline pkg/sanity/identity.go:41]: Also check if sc.ControllerConn != nil

Comment (carlory) [reply pkg/sanity/identity.go]: @xing-yang updated.

**Pull request merged**